lib/components/narrative/trip-tools.js

```diff
@@ -218,15 +216,11 @@ const TripTools = ({
         }
         break
       case 'START_OVER':
-        const startOverUrl = startOver(
-          reactRouterConfig?.basename,
-          location?.search
-        )
         buttonComponents.push(
           <LinkButton
             Icon={Undo}
+            onClick={() => startOverFromInitialUrl()}
```

Comment:
Can you just pass the function reference?

Reply:
I tried but it didn't like it for some reason. Something with the import?